raiden/blockchain/abi.py

```diff
@@ -70,6 +78,18 @@ def get_eventname_types(event_description):
     return name, encode_types
 
 
+#  Initially, I wanted to use solc's AST output, but that thing is so crazy that parsing
+#    it would require writing a new module to get just one solidity variable.
+#  So I opted for good ol' regex..
+def parse_contract_version(contract_file):
+    contract_file = get_contract_path(contract_file)
+    with open(contract_file, 'r') as original:
+        for line in original.readlines():
+            match = re.match(CONTRACT_VERSION_RE, line)
```

Comment:
Compile the RE somewhere outside and provide it as an argument to this function.